en/docs/trust/wso2-public-cloud-data-protection-faq.md

```diff
@@ -0,0 +1,395 @@
+
+### What is the legal basis for WSO2 cross-border data transfers?
+WSO2 operates globally and may transfer, store, access, or process subscribers' data across its affiliates and authorized sub-processors to provide the subscription and ancillary services.
+
+For transfers of personal data originating from the EU, WSO2 relies on valid legal mechanisms, including adequacy decisions or the EU Standard Contractual Clauses (2021). WSO2 ensures that all cross-border data transfers from the EU comply with GDPR requirements.
+
+WSO2 employees across WSO2 subsidiaries and affiliate companies access personal data collected by WSO2 solely to provide subscription and professional services for the subscriber. This includes ancillary operational activities related to subscription and professional services, such as account management and invoicing. Additionally, data collected from visitors to the WSO2 websites are accessed for website-related functionalities and services.
+
+This data will be accessed exclusively by WSO2 employees and the authorized sub-processors involved in the activities mentioned above.
+
+## Data Access
+
+### Who has access to subscriber data?
+* The Site Reliability Engineering (SRE) team would have access to the public cloud deployments to perform maintenance, administration, troubleshooting, and support. Only the SRE team is granted access to the deployments and their data. Other teams would need to go through service requests and change requests to gain deployment-related data access if required.
+* The Customer Reliability Engineering (CRE) team, which would interact with subscribers to support queries and issues, would have access to subscriber information on our support ticketing system and log data.
+* The Security Operations Center (SOC) and security team would have access to log data to monitor and respond to security events and incidents.
+* WSO2 account managers would have access to the support ticketing system and CRM for account management.
+
+### How do WSO2 employees access the cloud infrastructure?
+WSO2 uses Azure Privileged Identity Management (PIM) to control access to Azure resources. SRE team members must activate their eligible roles and submit requests for access. The team consists of specialized sub-teams, like Asgardeo-SRE and Choreo-SRE, ensuring no shared access across deployments. WSO2 also employs VPN connections with one-time passwords (OTPs) and uses Bastion instances for administrative tasks.
+
+### From what locations do you access data?
+Following are the locations where our teams are residing:
+* SRE - Sri Lanka, India, Brazil.
+* CRE - Sri Lanka, India, Australia, Brazil, Spain, UK, USA.
+* Security - Sri Lanka.
+* Account Management - Sri Lanka, USA, UK, Germany, Spain, Australia, India, Brazil, Dubai, and Singapore.
+
+## Technical and Organizational Controls
```

Comment:
Sub headings under this section are not matching with other sections. Other sections have questions as subheadings while this section is not directly questions? WDYT

Reply:
In this section, this is a main section and "Data Encryption" only has sub-sections that should be relected as questions. Current format is fine.